volatility3/framework/plugins/linux/sockstat.py

```diff
@@ -22,7 +22,7 @@ class SockHandlers(interfaces.configuration.VersionableInterface):
 
     _required_framework_version = (2, 0, 0)
 
-    _version = (2, 0, 0)
+    _version = (1, 0, 0)
```

Comment:
It's not a good idea to lower a version number just to correct a mistake.  I'd live with the increased version number and just move on from there...

Reply:
Reverted back to 2.0.0 and updated version requirement in `Sockscan` plugin class to match.